src/ng-fab-reset-form-on-d.spec.js

```diff
@@ -35,12 +37,14 @@ describe('a button with ngFabResetFormOn-directive', function ()
             form = scope.testForm;
             inputEl = angular.element(formEl.children()[0]);
             messageContainer = angular.element(formEl.children()[1]);
-            resetBtnEl = angular.element(formEl.children()[2]);
+            selectEl = angular.element(formEl.children()[2]);
+            resetBtnEl = angular.element(formEl.children()[4]);
```

Comment:
The third element is messageContainer for the select so no, it should't,

However, I updated this spec because the code coverage droped, but since I removed code duplication in the directive, these changes are not required. Do you want me to revert them?